### PR TITLE
Change element names to use dashes instead of underscores

### PR DESCRIPTION
**Pull request by nwalters512**
This changes all element names to use dashes instead of underscores in their name - this is more consistent with how Web Components name things, and editor syntax highlighting will finally work for questions 🙏

This change is so far backwards-compatible - I've aliased `***_***_***` to `***-***-***` for all core elements. I'm leaving courses to do what they want with their elements.

### CODE REVIEW

File: elements/pl-checkbox/pl-checkbox.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def prepare(element_html, element_index, data):
     incorrect_answers = []
     index = 0
     for child in element:
-        if child.tag == 'pl_answer':
+        if child.tag in ['pl-answer', 'pl_answer']:
```
Comment (mwest1066) [inline elements/pl-checkbox/pl-checkbox.py]: I think you are writing python :-) I believe JS needs [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes), not [in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) for arrays.

File: elements/pl-threejs/pl-threejs.py [hunk 2]
```diff
@@ -38,8 +38,8 @@ def get_objects(element, data):
     obj_list = []
 
     for child in element:
-        is_stl = (child.tag == 'pl_threejs_stl')
-        is_txt = (child.tag == 'pl_threejs_txt')
+        is_stl = (child.tag in ['pl-threejs-stl', 'pl_threejs_stl'])
+        is_txt = (child.tag == ['pl-threejs-txt', 'pl_threejs_txt'])
```
Comment (mwest1066) [inline elements/pl-threejs/pl-threejs.py]: Both wrong, for different reasons :-)

File: question-servers/freeform.js [hunk 3]
```diff
@@ -699,29 +707,21 @@ module.exports = {
...
-                        let isCourseElement = false;
-                        if (_.has(context.course_elements, elementName)) {
-                            elementDependencies = context.course_elements[elementName].dependencies || {};
-                            isCourseElement = true;
-                        } else {
-                            elementDependencies = coreElementsCache[elementName].dependencies || {};
-                        }
-
-                        elementDependencies = _.cloneDeep(elementDependencies);
+                        let resolvedElement = module.exports.resolveElement(elementName, context);
```
Comment (mwest1066) [inline question-servers/freeform.js]: Could throw?

File: elements/pl-checkbox/pl-checkbox.py (hunk 1 above)
Comment (nwalters512) [reply elements/pl-checkbox/pl-checkbox.py]: This is python though, right? I'm confused 😂

Comment (mwest1066) [reply elements/pl-checkbox/pl-checkbox.py]: Hahahaha! I was the most confused of all :-)

File: elements/pl-threejs/pl-threejs.py (hunk 2 above)
Comment (mwest1066) [reply elements/pl-threejs/pl-threejs.py]: Only one wrong, actually :-)

**Pull request merged**